kong/plugins/datadog/statsd_logger.lua

```diff
@@ -19,10 +19,16 @@ local stat_types = {
 local statsd_mt = {}
 statsd_mt.__index = statsd_mt
 
+local env_datadog_agent_host = os.getenv 'KONG_DATADOG_AGENT_HOST'
+local env_datadog_agent_port = os.getenv 'KONG_DATADOG_AGENT_PORT'
```

Comment:
```suggestion
local env_datadog_agent_port = tonumber(os.getenv 'KONG_DATADOG_AGENT_PORT')
```

Reply:
I think I should wait to convert the type until after confirming the environment variable is set.  I'm not sure what happens if I called `tonumber` on `nil`

Reply:
you're right, it should be `tonumber(os.getenv('KONG_DATADOG_AGENT_PORT') or "")`